src/main/java/com/nerodesk/om/aws/AwsDoc.java

```diff
@@ -120,6 +120,11 @@ public void read(@NotNull final OutputStream output) throws IOException {
     @Override
     public void write(final InputStream input, final long size)
         throws IOException {
+        if (size == 0) {
```

Comment:
@dmzaytsev Maybe, it would be better to have a decorator ```SafeDoc``` to check the size and avoid code duplication?